test/standalone/test-debuglet.js

```diff
@@ -44,25 +43,26 @@ var errorBp = {
   location: { path: 'fixtures/foo.js', line: 2 }
 };
 var fakeDebug = {
-  request: request
+  request: request // avoid authing.
```

Comment:
maybe it's just me but I don't know what this comment is referring to in this context